### PR TITLE
Migrate the project to Java Telegram Bot API

### PR DESCRIPTION
**Pull request by rob93c**
The scope of this pull request is to move the project from [TelegramBots](https://github.com/rubenlagus/TelegramBots) to [Java Telegram Bot API](https://github.com/pengrad/java-telegram-bot-api), which is up to date with Telegram's new features.

### CODE REVIEW

File: src/main/java/com/github/stickerifier/stickerify/bot/Stickerify.java [hunk 1]
```diff
@@ -30,26 +30,26 @@
...
-	}
+		super(BOT_TOKEN);
 
-	@Override
-	public String getBotUsername() {
-		return Stickerify.class.getSimpleName();
+		setUpdatesListener(updates -> {
+			updates.forEach(this::handleUpdate);
+			return UpdatesListener.CONFIRMED_UPDATES_ALL;
+		}, e -> LOGGER.error("There was an unexpected failure: " + e.getMessage()));
```
Comment (MartelliEnrico) [inline src/main/java/com/github/stickerifier/stickerify/bot/Stickerify.java:47]:
```suggestion
		}, e -> LOGGER.error("There was an unexpected failure: {}", e.getMessage()));
```

File: src/main/java/com/github/stickerifier/stickerify/bot/Stickerify.java [hunk 2]
```diff
@@ -30,26 +30,26 @@
...
 	}
 
-	@Override
-	public void onUpdateReceived(Update update) {
-		if (update.hasMessage()) {
-			TelegramRequest request = new TelegramRequest(update.getMessage());
+	private void handleUpdate(Update update) {
+		if (update.message() != null) {
+			TelegramRequest request = new TelegramRequest(update.message());
```
Comment (MartelliEnrico) [inline src/main/java/com/github/stickerifier/stickerify/bot/Stickerify.java:52]:
```suggestion
			var request = new TelegramRequest(update.message());
```

File: src/main/java/com/github/stickerifier/stickerify/bot/Stickerify.java [hunk 3]
```diff
@@ -97,32 +96,35 @@ private void answerFile(TelegramRequest request) {
 	}
 
 	private File retrieveFile(String fileId) throws TelegramApiException {
-		GetFile getFile = new GetFile(fileId);
+		var file = execute(new GetFile(fileId)).file();
```
Comment (MartelliEnrico) [inline src/main/java/com/github/stickerifier/stickerify/bot/Stickerify.java:99]: Shouldn't we check for `.isOk()` here as well?

**Review by MartelliEnrico — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/stickerifier/stickerify/bot/Stickerify.java (hunk 3 above)
Comment (rob93c) [reply src/main/java/com/github/stickerifier/stickerify/bot/Stickerify.java:99]: Yep, you are right! I will amend the code accordingly

**Review by MartelliEnrico — APPROVED** [review]
LGTM

**Pull request merged**